portfolio/src/main/webapp/css/reviews.css

```diff
@@ -0,0 +1,125 @@
+.section {
+  text-align: center;
+  padding: 100px 160px;
+}
+
+.section-dark {
+  background-color: black;
+  color: white;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+input[type=text], select, textarea {
+  width: 100%;
```

Comment:
Super nit, but these tend to be alphabetized: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

so...
border:
border-radius:
padding:
resize:

etc,